### ISSUE TITLE
Section 3.1.1 unclear

### ISSUE THREAD

**Issue by thomas-fossati** [yaronf/I-D#420]
_Comment by @evyncke_

Perhaps because I am not a native English speaker, but I find this sentence hard to parse:

~~~ 
      Even if a TLS
      implementation defaults to TLS 1.3, as long as it supports TLS 1.2
      it MUST follow all the recommendations in this document.
~~~

**Comment by stpeter** [comment]
Via email I suggested the following:

> To the extent that an implementation supports TLS 1.2 (even if it defaults to TLS 1.3), it MUST follow the recommendations regarding TLS 1.2 specified in this document.
